source/2015-03-09-v1-4-1-is-out.html.markdown

```diff
@@ -0,0 +1,11 @@
+---
+title: Database Cleaner v1.4.1 is out!
+date: 2015-03-09
+tags: release
+---
+
+# Database Cleaner v1.4.1 is out!
+
+The latest version contains a few fixes. Please check out: https://github.com/DatabaseCleaner/database_cleaner/blob/master/History.rdoc#141-2015-03-09
```

Comment:
Rather than linking to the rdoc, it'd be nicer to surmise the changes here, preferably with a bit more history than is currently present in the doc!